app/decorators/decidim/admin/users_controller_decorator.rb

```diff
@@ -19,8 +20,76 @@ def user
     @user ||= original_collection.find(params[:id])
   end
 
+  def show
```

Comment:
I think that the indentation is not correct, making it very difficult to read. Can you check it please?

Reply:
I just check it and it seems fine in my editor